atlas/AtlasMl/atlasml/ml/MLPipelines/PipelineWorkflows.py

```
@@ -0,0 +1,278 @@
+import uuid
+import numpy as np
+from atlasml.clients.weaviate import get_weaviate_client, CollectionNames, WeaviateClient
```

Ruff (0.11.9)

`atlasml.clients.weaviate.WeaviateClient` imported but unused

Remove unused import: `atlasml.clients.weaviate.WeaviateClient`

(F401)